File: Task-03/prime.rb
def is_prime(num)
  return false if num < 2
  (2..Math.sqrt(num).to_i).each do |i|
    return false if num % i == 0
  end
  true
end

print "Enter a number: "
n = gets.chomp.to_i
primes = (2..n).select { |num| is_prime(num) }
puts "Prime numbers up to #{n} are: #{primes}"

File: Task-03/prime.js
function isPrime(num) {
    if (num < 2) return false;
    for (let i = 2; i <= Math.sqrt(num); i++) {
        if (num % i === 0) return false;
    }
    return true;
}

const n = parseInt(prompt("Enter a number:"));
const primes = [];
for (let i = 2; i <= n; i++) {
    if (isPrime(i)) {
        primes.push(i);
    }
}
console.log(`Prime numbers up to ${n} are: ${primes}`);

